plugin/storage/es/dependencystore/storage.go

```diff
@@ -121,17 +122,12 @@ func buildTSQuery(endTs time.Time, lookback time.Duration) elastic.Query {
 	return elastic.NewRangeQuery("timestamp").Gte(endTs.Add(-lookback)).Lte(endTs)
 }
 
-func (s *DependencyStore) getIndices(ts time.Time, lookback time.Duration) []string {
+func getIndices(ts time.Time, lookback time.Duration) []string {
 	var indices []string
 	indices = append(indices, indexName(ts))
 	for lookback > 0 {
```

Comment:
right, but we're still telling the query service to search in the previous index even though we don't have to